nrf_dm/include/nrf_dm.h

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file nrf_dm.h
+ *
+ * @defgroup nrf_dm nRF Distance Measurement interface
+ *
+ * Interface for the Nordic distance measurement library.
+ * @{
+ */
+
+#ifndef __NRF_DM_H_
+#define __NRF_DM_H_
+
+#include <stdint.h>
+#include "nrf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Quality of the procedure
+ */
+typedef enum {
+	NRF_DM_QUALITY_OK = 0,
+	NRF_DM_QUALITY_POOR = 1,
+	NRF_DM_QUALITY_DO_NOT_USE = 2,
+	NRF_DM_QUALITY_CRC_FAIL = 3,
```

Comment:
Consider adding structure field comments, for consistency.

Reply:
do you have a suggestion for the information to add?

Reply:
Meybe:
```
/* Good measurement quality */
/* Poor measurement quality */
/* Measurement not for use */
```
I added this suggestion because almost all the structure fields in this file have comments.

Reply:
Is this not already visible in the enum names?

Reply:
Yes this is visible. It's just a suggestion.

Reply:
ok, I will leave this as is, for now.